arch.mk

```diff
@@ -23,10 +23,9 @@ arch-setup:
 arch-dokku: arch-setup
 	echo "-> Update package sums, create metadata file and test the build of the package"
 ifeq ($(DOKKU_VERSION),master)
-	git describe --tags > /tmp/VERSION
+	git describe --tags > /tmp/version
 else
-	echo $(DOKKU_VERSION) > /tmp/VERSION
+	echo $(DOKKU_VERSION) > /tmp/version
```

Comment:
Lets avoid changing the case. Keep it upper-case for now.

Reply:
reverted